modules/031-linstor/monitoring/prometheus-rules/drbd-devices.yaml

```diff
@@ -0,0 +1,158 @@
+- name: kubernetes.drbd.device_state
+  rules:
+    - alert: LinstorVolumeIsNotHealthy
+      expr: linstor_volume_state != 1 and linstor_volume_state != 4
```

Comment:
It would be best if you thought about cutting some labels, e.g., by applying the `max by` filter. There are labels like `instance`, that may change after pod restart. For some alertmanagers, trigger names and all labels compose a unique hash id for the incident.